### PR TITLE
Updated Aura with secret service support

### LINKED ISSUE #569: Update Aura to 2023.9.1

### PR DESCRIPTION
**Pull request by fsobolev**
Closes #569
Depends on https://github.com/NickvisionApps/Aura/pull/18
Depends in https://github.com/NickvisionApps/Aura/pull/23

TODO:
- [x] Update csproj to reference new version of Aura
- [x] Fix creating Parabolic keyring when keyring is locked
- [x] Update po(t)

### CODE REVIEW

**Comment by fsobolev** [comment]
Forgot to update flatpak sources, and also need to check whether flatpak requires any new permissions, will build locally to test

**Comment by nlogozzo** [comment]
> Forgot to update flatpak sources, and also need to check whether flatpak requires any new permissions, will build locally to test

Probably need `"--talk-name=org.freedesktop.secrets"`

File: flatpak/org.nickvision.tubeconverter.json [hunk 1]
```diff
@@ -171,6 +172,7 @@
                 }
             ]
         },
+        "libsecret/libsecret.json",
```
Comment (nlogozzo) [inline flatpak/org.nickvision.tubeconverter.json:175]: Why couldn't you just put the libsecret module here....no need to keep it in another file

**Review by nlogozzo — CHANGES_REQUESTED** [review]
Another question: What happens to Keyrings created with old Parabolic and then opened in this new Parabolic? Will they still be asked for the password everytime to open the Keyring? Or the first time they open the Keyring it will get stored in libsecret and they won't be asked again?

File: flatpak/org.nickvision.tubeconverter.json (hunk 1 above)
Comment (fsobolev) [reply flatpak/org.nickvision.tubeconverter.json:175]: Indeed, sorry, I first used libsecret from shared-modules and so placed it in separate folder (the same as with intltool), I will move it.

**Comment by fsobolev** [comment]
> Another question: What happens to Keyrings created with old Parabolic and then opened in this new Parabolic? Will they still be asked for the password everytime to open the Keyring? Or the first time they open the Keyring it will get stored in libsecret and they won't be asked again?

I didn't test it myself, but I expect that a new random password will be generated and saved in system keyring (only at first start) but because it will not match the keyring's password the user will be prompted every time. There are 2 ways to migrate: either remove keyring and create new one, or save the keyring passord to system keyring, this can be easily done using Seahorse, I want to write a small guide as soon as the beta releases.

File: flatpak/org.nickvision.tubeconverter.json [hunk 2]
```diff
@@ -172,7 +172,7 @@
                 }
             ]
         },
-        "libsecret/libsecret.json",
+        "libsecret.json",
```
Comment (nlogozzo) [inline flatpak/org.nickvision.tubeconverter.json:175]: What I meant it why have a separate `libsecret.json` file all together? Just paste it inside the manifest itself...

**Review by nlogozzo — CHANGES_REQUESTED** [review]

File: flatpak/org.nickvision.tubeconverter.json (hunk 2 above)
Comment (fsobolev) [reply flatpak/org.nickvision.tubeconverter.json:175]: Ah, okay

**Comment by nlogozzo** [comment]
Button to Docs in PasswordDialog:
image

**Review by nlogozzo — APPROVED** [review]

**Comment by nlogozzo** [comment]
@soumyaDghosh is there anything that needs to be added to the snap yml for `libsecret` use?

**Comment by soumyaDghosh** [comment]
It's in gnome-sdk

**Comment by fsobolev** [comment]
> It's in gnome-sdk

Please check manually that it works (simply enabling keyring is enough), there's some weird thing in sandboxed environment so we have to build libsecret with `-Dgcrypt=false` even tho libsecret is in GNOME Platform in flatpak too.

**Comment by fsobolev** [comment]
> Please check manually that it works (simply enabling keyring is enough), there's some weird thing in sandboxed environment so we have to build libsecret with -Dgcrypt=false even tho libsecret is in GNOME Platform in flatpak too.
>
> And [pasword-manager-service](https://snapcraft.io/docs/password-manager-service-interface) plug seems to be needed anyway.

Just like I thought, it doesn't work in Snap even with `password-manager-service`, libsecret should be built with `-Dgcrypt=false`

**Pull request merged**